src/main/java/rotateit/domain/github/label/event/UnknownReviewActionEvent.java

```diff
@@ -1,9 +1,15 @@
 package rotateit.domain.github.label.event;
 
 import org.springframework.context.ApplicationEvent;
+import rotateit.domain.github.label.LabelPayload;
 
 public class UnknownReviewActionEvent extends ApplicationEvent {
-    public UnknownReviewActionEvent(Object source) {
-        super(source);
+    public UnknownReviewActionEvent(LabelPayload labelPayload) {
+        super(labelPayload);
+    }
+
+    @Override
+    public LabelPayload getSource() {
+        return (LabelPayload) super.getSource();
     }
```

Comment:
You can use generics so that casting wouldn't be required here and in other places.